dashboard/src/components/users/UsersTable.vue

```diff
@@ -60,17 +60,26 @@ limitations under the License.
           </b-button-group>
         </template>
         <template v-slot:cell(totalPoints)="data">
-          <div class="row" :data-cy="`usr_progress-${data.item.userId}`">
-            <div class="col-auto">
-              <span class="font-weight-bold text-primary" data-cy="progressPercent">{{ calcPercent(data.value) }}%</span>
+          <div :data-cy="`usr_progress-${data.item.userId}`">
+            <div class="row">
+              <div class="col-auto">
+                <span class="font-weight-bold text-primary" data-cy="progressPercent">{{ calcPercent(data.value) }}%</span>
```

Comment:
span showing percent could use a better aria-label for screenreader users

Reply:
:+1: